src/imports/types.js

```diff
@@ -10,9 +10,11 @@ export type UserRecord = {
 export interface StorageAPI {
     addUser(user: UserRecord): Promise<boolean>,
     updateUser(user: UserRecord): Promise<boolean>,
-    deleteUser(user: UserRecord): Promise<boolean>
+    deleteUser(user: UserRecord): Promise<boolean>,
+    storeOTP(user: UserRecord, otp: number): Promise<boolean>
```

Comment:
I don't think we need storeOTP/getOTP, simpler to just use updateUser and add the otp details to the UserRecord.
the UserRecord is available in express requests.

Reply:
The idea of having a different gundb document was to not populate the UserRecord with temp data. But that data can be optional in the UserRecord and can be removed after confirming. So I'll change it. Thanks!